beacontools/scanner.py

```diff
@@ -157,8 +152,8 @@ def toggle_scan(self, enable, filter_duplicates=False):
             enable: boolean value to enable (True) or disable (False) scanner
             filter_duplicates: boolean value to enable/disable filter, that
                 omits duplicated packets"""
-        command = struct.pack(">BB", enable, filter_duplicates)
-        self.bluez.hci_send_cmd(self.socket, OGF_LE_CTL, OCF_LE_SET_SCAN_ENABLE, command)
+        command = struct.pack("BB", enable, filter_duplicates)
```

Comment:
why did you change this?

Reply:
Here, you do not need any endianness specifier, as single byte values have no endianness